environments/common/inventory/group_vars/all/grafana.yml

```diff
@@ -74,12 +75,15 @@ grafana_datasources:
       tlsSkipVerify: true
       database: filebeat-*
       timeField: "@timestamp"
-    flavor: opensearch
+      # Have to set flavor and version, but ansible/roles/opensearch/templates/opensearch.yml.j2 fakes version for filebeat
+      # so need to set to fake version here:
+      version: '7.10.2'
+      flavor: elasticsearch
     editable: true
     # readOnly: false
 
 grafana_plugins:
-  - grafana-opensearch-datasource
+  - grafana-opensearch-datasource 2.8.1
```

Comment:
Does it make sense to expose this as its own variable: `grafana_opensearch_datasource_version` or something?

Reply:
Personally, I don't think so. It'd just more templating and vars in common, when really we want less there, without actually adding making it more customisable.